dev/SapphireTestReporter.php

```diff
@@ -1,6 +1,10 @@
 <?php
-if(!class_exists('PHPUnit_Framework_TestResult', false)) require_once 'PHPUnit/Framework/TestResult.php';
-if(!class_exists('PHPUnit_Framework_TestListener', false)) require_once 'PHPUnit/Framework/TestListener.php';
+if(!class_exists('PHPUnit_Framework_TestResult', false)) {
```

Comment:
phpunit could have been installed globally, instead of via composer. In which case, it could have been declared in the global include paths, couldn't it?